subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -115,6 +115,52 @@ static void gatt_proxy_status(struct bt_mesh_model *model,
 	state_status_u8(model, ctx, buf, OP_GATT_PROXY_STATUS);
 }
 
+static void transmit_status(struct bt_mesh_model *model,
+			    struct bt_mesh_msg_ctx *ctx,
+			    struct net_buf_simple *buf)
+{
+	state_status_u8(model, ctx, buf, OP_NET_TRANSMIT_STATUS);
+}
+
+struct krp_param {
+	uint8_t *status;
+	uint16_t net_idx;
+	uint8_t *phase;
+};
+
+static void krp_status(struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
+		       struct net_buf_simple *buf)
+{
+	struct krp_param *param;
+	uint16_t net_idx;
+	uint8_t status, phase;
+
+	BT_DBG("net_idx 0x%04x app_idx 0x%04x src 0x%04x len %u: %s",
+	       ctx->net_idx, ctx->app_idx, ctx->addr, buf->len,
+	       bt_hex(buf->data, buf->len));
+
+	if (!bt_mesh_msg_ack_ctx_match(&cli->ack_ctx, OP_KRP_STATUS, ctx->addr,
+				       (void **)&param)) {
+		return;
+	}
+
+	status = net_buf_simple_pull_u8(buf);
+	net_idx = net_buf_simple_pull_le16(buf) & 0xfff;
+	phase = net_buf_simple_pull_u8(buf);
+
+	if (param->net_idx != net_idx) {
+		BT_WARN("Net Key Status key index does not match");
```

Comment:
I believe this practice of warning about non-matching has stopped, as it just creates noise